packages/aragon-wrapper/src/index.js

```diff
@@ -439,6 +439,109 @@ export default class Aragon {
       )
     )
 
+    this.installedRepos = new ReplaySubject(1).pipe(
```

Comment:
`initApps()` is becoming way too much of a beast.

I'd like to start exploring how we can compartmentalize this into its own module; one idea:

- Perhaps this module could expose one big `init()` function (or multiple smaller `initX()` functions), that returns these observables we attach onto `this`
- This module could be built up of multiple smaller methods that don't rely on the global `this` state